.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,18 +3,3 @@ I hereby agree to the terms of the CLA available at: https://databend.rs/dev/pol
 ## Summary
 
 Summary about this PR
-
-## Changelog
-
-- New Feature
-- Bug Fix
-- Improvement
-- Performance Improvement
-- Build/Testing/CI
-- Documentation
-- Not for changelog (changelog entry is not required)
-
-## Related Issues
-
-Fixes #issue
```

Comment:
I think it can be kept.

Reply:
Don't make sense to me, how about adding under summary?